### PR TITLE
API call made and reformatted (serialized)

### PR DESCRIPTION
**Pull request by TravisGM92**
### Participating Group Members:
* Travis McKinstry 
 
### Code Highlights:
   * Creates a service file where API call is made
   * Service file reformats the object and sends it back out
   * Path can take in additional parameters (such as an ID, location, etc.)
 
### Have Tests Been Added?
- [ ] No.
- [ ] Yes, but all tests are not passing. 
- [ ] Yes, and all are passing.
 
### Message/Questions for reviewer:

This will be a great template to use for when we work on the Google API call. We can just copy this exactly
 
### Issues: 
* Addresses # needing to make an API call
* Closes #
 
### Screenshots (if appropriate): 
*
 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [ ] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: boneyardproj/urls.py [hunk 1]
```diff
@@ -16,9 +16,12 @@
...
 from main_app import views
+from main_app import services
+from django.conf.urls import url
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('test/', views.home),
-    path('', views.index)
+    path('', views.index),
+    url(r'^api/(?P<pk>[0-9]+)', services.get_parks),
```
Comment (TravisGM92) [inline boneyardproj/urls.py:26]: I don't understand this part too much. I think the 'r'^api/...  is first the path (<website url>/api/) but the part after that is maybe a "allow for numbers or letters to come after". Not sure

File: main_app/services.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from django.http import JsonResponse
+import requests
+
+def get_parks(self, pk):
+    info = requests.get('https://jsonplaceholder.typicode.com/todos/1').json()
+    data = {
+    'name': info['title']
+    }
+    return JsonResponse(data)
```
Comment (TravisGM92) [inline main_app/services.py:9]: We'll likely need to pull some of this out and put it into a Serializer class. Just not sure how to do that yet.

**Comment by TravisGM92** [comment]
I realized that since we haven't been PR-ing to main, making this PR to development won't do us any good. So we can just disregard that for now. Submit all PR's to main

File: boneyardproj/urls.py (hunk 1 above)
Comment (robertorodriguez12) [reply boneyardproj/urls.py:26]: Good work here!

**Review by robertorodriguez12 — APPROVED** [review]

File: main_app/services.py (hunk 2 above)
Comment (judithpillado) [reply main_app/services.py:9]: Looks great!

**Review by judithpillado — APPROVED** [review]

**Comment by judithpillado** [comment]
Thank you for doing this Travis!

**Pull request merged**